Makefile

```diff
@@ -349,3 +349,24 @@ integration-test:
 	else\
 		$(GINKGO) -v $(TEST_ARGS) --fail-fast --focus-file=$(TEST_FILE) test/integration;\
 	fi
+
+#hosted
+ADDON_CONTROLLER = $(PWD)/governance-policy-addon-controller
+
+.PHONY: kind-bootstrap-hosted
+kind-bootstrap-hosted: $(ADDON_CONTROLLER) kind-install-hosted install-crds install-resources kind-deploy-policy-framework kind-deploy-policy-controllers
+
+.PHONY: install-hosted
+kind-install-hosted: 
+	@cd ./governance-policy-addon-controller && KIND_VERSION=$(KIND_VERSION) HOSTED_MODE=true ./build/manage-clusters.sh
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig ./kubeconfig_hub
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig-internal ./kubeconfig_hub_internal
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl2.kubeconfig ./kubeconfig_managed
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl2.kubeconfig-internal ./kubeconfig_managed_internal
+
+$(ADDON_CONTROLLER):
+	git clone --depth=1 -b $(RELEASE_BRANCH) https://github.com/stolostron/governance-policy-addon-controller.git
+
+delete-hosted: 
+	@cd governance-policy-addon-controller && make kind-bootstrap-delete-clusters 
+	@rm kubeconfig_hub kubeconfig_managed kubeconfig_hub_internal kubeconfig_managed_internal 
```

Comment:
Rather than adding the directory target to the bootstrap list, I'd prefer it be a prerequisite for the install-hosted target (that way, install-hosted can be run independently if desired):

```suggestion
ADDON_CONTROLLER = $(PWD)/governance-policy-addon-controller

.PHONY: kind-bootstrap-hosted
kind-bootstrap-hosted: kind-install-hosted install-crds install-resources kind-deploy-policy-framework kind-deploy-policy-controllers

.PHONY: install-hosted
kind-install-hosted: $(ADDON_CONTROLLER)
	@cd $(ADDON_CONTROLLER) && KIND_VERSION=$(KIND_VERSION) HOSTED_MODE=true ./build/manage-clusters.sh
	@cp $(ADDON_CONTROLLER)/policy-addon-ctrl1.kubeconfig ./kubeconfig_hub
	@cp $(ADDON_CONTROLLER)/policy-addon-ctrl1.kubeconfig-internal ./kubeconfig_hub_internal
	@cp $(ADDON_CONTROLLER)/policy-addon-ctrl2.kubeconfig ./kubeconfig_managed
	@cp $(ADDON_CONTROLLER)/policy-addon-ctrl2.kubeconfig-internal ./kubeconfig_managed_internal

$(ADDON_CONTROLLER):
	git clone --depth=1 -b $(RELEASE_BRANCH) https://github.com/stolostron/governance-policy-addon-controller.git

delete-hosted: 
	@cd $(ADDON_CONTROLLER) && make kind-bootstrap-delete-clusters
	@rm kubeconfig_hub kubeconfig_managed kubeconfig_hub_internal kubeconfig_managed_internal

```